packages/astro-og-canvas/README.md

```diff
@@ -2,7 +2,7 @@
 
 This package provides utlities to generate OpenGraph images for the pages on your Astro site.
 
-## Installation
+## Installation[^1]
```

Comment:
I think the footnote format may not be the clearest as it forces a reader to jump around the page.

I’d probably adding a comment immediately below the install instructions, so we have something like:

````md
## Installation

```shell
npm i astro-og-canvas
```

**Using `pnpm`?** `pnpm` users will also need to install `canvaskit-wasm` as a direct dependency:

```shell
pnpm i canvaskit-wasm
```
````

Reply:
Done!